app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
```

Comment:
No, maybe my first comment wasn't clear enough.

> I don't think we should do this.

I wrote this because I don't think that we should override the button height. The height of the buttons depends on the font-size, line-height and so on. Whenever someone changes one of those values the height you set here would be wrong.

Why did you add this line?